Faraday_Proxy_Tools/FaradayIO/deviceconfig.py

```diff
@@ -33,8 +34,12 @@ def __init__(self):
         # Definitions
```

Comment:
In d573b94b77d0094175043109841e5221afc35f46 I have added checking for:

* Callsign length max
* Callsign ID between 0-255
* GPIO default 3, 4, and 5 between 0-255